xml/System.Reflection/AssemblyFlags.xml

```diff
@@ -49,7 +49,7 @@
       </ReturnValue>
```

Comment:
Is a description available for the AssemblyFlags type itself?

Reply:
There isn't one. Otherwise, the tool would've caught it and ported it: [src/System.Reflection.Metadata/src/System/Reflection/System.Reflection.cs#L193](https://github.com/dotnet/corefx/blob/d3911035f2ba3eb5c44310342cc1d654e42aa316/src/System.Reflection.Metadata/src/System/Reflection/System.Reflection.cs#L193)

Reply:
@steveharter @GrabYourPitchforks would you mind providing a description for this enum?